frontend/src/views/producer/dashboard/edition-form/TicketCatalogGroup.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import Button from '../../../../components/button';
+import { InputField } from '../../../../components/form';
+import { TicketCatalog } from '../../../../models/event';
```

Comment:
Would it be possible to use absolute path instead of relative? eg. `from '~/models/event'; `